app/login/accessError/page.tsx

```diff
@@ -0,0 +1,12 @@
+'use client';
+
+// import Link from 'next/link';
+import AccessDenied from '../../../src/components/Auth/access-denied';
```

Comment:
we will need to update this once we have full @/module links

Reply:
it should work already

George, this can be reduced for `@/components` if you export from the index files, there are a couple examples you can reference

Reply:
thanks, I have changed to @/components format